symfony/client/src/orangehrmPerformancePlugin/pages/KpiEdit.vue

```diff
@@ -0,0 +1,233 @@
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-card-container">
+      <oxd-text class="orangehrm-main-title"
+        >Edit Key Performance Indicator</oxd-text
+      >
+      <oxd-divider />
+
+      <oxd-form :loading="isLoading" @submitValid="onSave">
+        <oxd-form-row>
+          <oxd-grid :cols="2" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                v-model="kpi.title"
+                label="Key Performance Indicator"
+                required
+                :rules="rules.title"
+                autcomplete="off"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <jobtitle-dropdown
+                v-model="kpi.jobTitle"
+                required
+                :rules="rules.jobTitle"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="4" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                v-model.number="kpi.minRate"
+                label="Minimum Rating"
+                required
+                autcomplete="off"
+                :rules="rules.minRate"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <oxd-input-field
+                v-model.number="kpi.maxRate"
+                label="Maximum Rating"
+                required
+                autcomplete="off"
+                :rules="rules.maxRate"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <div class="orangehrm-module-field-row">
+                <oxd-text tag="p" class="orangehrm-module-field-label">
+                  Make Default Scale
+                </oxd-text>
+                <oxd-switch-input v-model="kpi.isDefault" />
+              </div>
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+
+        <oxd-divider />
+        <br />
+        <oxd-form-actions>
+          <required-text />
+          <oxd-button
+            type="button"
+            display-type="ghost"
+            label="Cancel"
+            @click="onCancel"
+          />
+          <submit-button />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import SwitchInput from '@ohrm/oxd/core/components/Input/SwitchInput';
+import {navigate} from '@ohrm/core/util/helper/navigation';
+import {
+  required,
+  digitsOnly,
+  shouldNotExceedCharLength,
+  minValueShouldBeLowerThanMaxValue,
+} from '@ohrm/core/util/validation/rules';
+import {APIService} from '@/core/util/services/api.service';
+import JobtitleDropdown from '@/orangehrmPerformancePlugin/components/JobtitleDropdown.vue';
+
+const initialKpi = {
+  title: '',
+  jobTitle: null,
+  minRate: null,
+  maxRate: null,
+  isDefault: false,
+};
+
+const rateError = v => {
+  if (digitsOnly(v)) {
```

Comment:
`digitsOnly()` return `true` if it's a digit and `string` if not. `string` will be taken as boolean `true` in by js `type coercion`. Better implicitly check the type in here to avoid unfavourable side effects.
```
if (digitsOnly(v)===true)
```

Reply:
Made the change